CONTRIBUTING.md

```diff
@@ -73,6 +73,23 @@ That's it! The plugin will run every time you commit any changes. If there are a
 pre-commit run --all-files --show-diff-on-failure
 ```
 
+
+ <details>
+  <summary>Hint for windows users</summary>
+  <br/>
+  On Windows, the python3 command is not recognized. This can lead to the error below.
```

Comment:
Windows users should use `py`

Reply:
Yes, but we cannot make such assumptions for pre commit hooks, they rely on the python3 command.

Reply:
#12371